plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/RemoteTableName.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.cloud.bigquery.TableId;
+import com.google.common.base.Joiner;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Qualified table name reported by the remote database.
```

Comment:
make the comment less generic... or maybe just remove

Reply:
I wish I could do without this class altogether.

I'll make the comment more detailed regarding case-sensitivity of BigQuery.

Reply:
my call for removal was about javadoc, not the class.